GNUmakefile

```
@@ -1,3 +1,5 @@
+SHELL=/bin/bash -o pipefail
```

```suggestion
SHELL=/usr/bin/env -S bash -o pipefail
```
for the NixOS/BSD users out there